suricata.yaml.in

```diff
@@ -139,6 +139,18 @@ outputs:
       enabled: @e_enable_evelog@
       filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
       filename: eve.json
+
+      # JSON flags
```

Comment:
I wonder if this is one that we should just document in the user guide instead of further bloating the yaml portion. Thoughts @jasonish @regit ?